monai/transforms/utility/array.py

```diff
@@ -1715,3 +1715,101 @@ def __call__(self, img: NdarrayOrTensor, meta_dict: Mapping | None = None) -> Nd
         if self._do_transform:
             img = self.filter(img)
         return img
+
+
+class CoordinateTransform(InvertibleTransform, Transform):
```

Comment:
I'm struggling with why this is a class. Is it specifically for converting points from "image" to "world" and vice versa or is it a base class for geometry? We don't have a base raster class and I'm not sure a base class can be defined that would cover all of the specific transforms cleanly. We have a functional layer for most of our operations these days; I'd argue that all baseline geometry should be defined at the functional level and wrapped by classes, rather than defined as classes.

Reply:
Closing this as we already discussed this in meetings. Please feel free to re-open it if there are questions @atbenmurray